verilog/formatting/tree_unwrapper.cc

```diff
@@ -516,6 +516,7 @@ void TreeUnwrapper::InterChildNodeHook(const SyntaxTreeNode& node) {
   switch (tag) {
     // TODO(fangism): cover all other major lists
     case NodeEnum::kPortDeclarationList:
+    case NodeEnum::kPortActualList:
```

Comment:
Thanks for this fix.  Can you also include some tests inside tree_unwrapper_test.cc that cover this directly, using some of the above test cases with comments?  Those tests verify that partitioning structure directly.